gems/sorbet-runtime/lib/types/compatibility_patches.rb

```diff
@@ -36,3 +36,27 @@ def observe!(method_name)
     end
   end
 end
+
+module T
+  module CompatibilityPatches
+    module MethodExtensions
+      def arity
+        arity = super # rubocop:disable PrisonGuard/NoArity
+        return arity if arity != -1 || self.is_a?(Proc)
+        sig = T::Private::Methods.signature_for_method(self)
+        sig ? sig.method.arity : arity # rubocop:disable PrisonGuard/NoArity
```

Comment:
```suggestion
        sig ? sig.method.arity : arity
```

Reply:
Now that this code isn't in our monorepo anymore, these rubocops in the `PrisonGuard` namespace don't exist, so there's no use in propagating them.